### PR TITLE
Copy edits for the JS “Rest parameters” article

### PR DESCRIPTION
**Pull request by sideshowbarker**

### CODE REVIEW

File: files/en-us/web/javascript/reference/functions/rest_parameters/index.html [hunk 1]
```diff
@@ -208,7 +203,7 @@ <h2 id="Browser_compatibility">Browser compatibility</h2>
 <h2 id="See_also">See also</h2>
 
 <ul>
- <li><a href="/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator" title="spread operator">Spread syntax</a> (also ‘<code>...</code>’)</li>
+ <li><a href="/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax" title="spread operator">Spread syntax</a> (also ‘<code>...</code>’)</li>
```
Comment (nschonni) [inline files/en-us/web/javascript/reference/functions/rest_parameters/index.html:206]:
```suggestion
 <li><a href="/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax">Spread syntax</a> (also ‘<code>...</code>’)</li>
```

Comment (sideshowbarker) [reply files/en-us/web/javascript/reference/functions/rest_parameters/index.html]: 👍 Thanks for catching that

Comment (nschonni) [reply files/en-us/web/javascript/reference/functions/rest_parameters/index.html:206]: Looks like it came back with the rebase

Comment (sideshowbarker) [reply files/en-us/web/javascript/reference/functions/rest_parameters/index.html:206]: > Looks like it came back with the rebase

Dang — thanks again for (re)catching it. Will (re)fix

File: files/en-us/web/javascript/reference/functions/rest_parameters/index.html [hunk 2]
```diff
@@ -40,19 +38,16 @@ <h2 id="Description">Description</h2>
...
- <li>There can be only one ...restParam.<br>
-  <code>   foo(...one, ...wrong, ...wrong)</code></li>
- <li>Rest parameter must be the last argument.<br>
-  <code>   foo(...wrong, arg2, arg3)</code><br>
-  <code>   foo(arg1, arg2, ...correct)</code></li>
- <li>Rest parameter can be <a href="/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment">destructured Arrays</a> (for advanced users only :).<br>
-  <code>   foo(arg1, ...[2,4,6])</code></li>
-</ul>
+ <p>A function definition can have only one <code>...</code><var>restParam</var>.</p>
+  <pre>foo(...one, ...wrong, ...wrong)</pre>
```
Comment (Elchi3) [inline files/en-us/web/javascript/reference/functions/rest_parameters/index.html:44]:
```suggestion
  <pre class="brush:js example-bad">foo(...one, ...wrong, ...wrong)</pre>
```

File: files/en-us/web/javascript/reference/functions/rest_parameters/index.html [hunk 3]
```diff
@@ -40,19 +38,16 @@ <h2 id="Description">Description</h2>
...
+ <p>The rest parameter must be the last parameter in the function
+ definition.</p>
+  <pre>foo(...wrong, arg2, arg3)</pre>
```
Comment (Elchi3) [inline files/en-us/web/javascript/reference/functions/rest_parameters/index.html:47]:
```suggestion
  <pre class="brush:js example-bad">foo(...wrong, arg2, arg3)</pre>
```

File: files/en-us/web/javascript/reference/functions/rest_parameters/index.html [hunk 4]
```diff
@@ -40,19 +38,16 @@ <h2 id="Description">Description</h2>
...
+  <pre>foo(arg1, arg2, ...correct)</pre>
```
Comment (Elchi3) [inline files/en-us/web/javascript/reference/functions/rest_parameters/index.html:48]:
```suggestion
  <pre class="brush:js example-good">foo(arg1, arg2, ...correct)</pre>
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Well done, nice cleaning. Just some formatting nits.

**Comment by Elchi3** [comment]
Looks like that rebase / merge conflict is still there or came back yet again?

**Comment by sideshowbarker** [comment]
> Looks like that rebase / merge conflict is still there or came back yet again?

Dunno exactly what, but will fix right now

**Comment by Elchi3** [comment]
Looks resolved now. However, the pre styling I proposed is gone?

**Review by Elchi3 — APPROVED** [review]
Awesome, thanks! :)

**Pull request merged**